docs/examples/micropayment.rst

```diff
@@ -346,11 +346,11 @@ The full contract
         address payable public recipient;   // The account receiving the payments.
         uint256 public expiration;  // Timeout in case the recipient never closes.
 
-        constructor (address payable _recipient, uint256 duration)
+        constructor (address payable recipient_, uint256 duration)
```

Comment:
Maybe it would be better to make a larger change to the parameter name to distinguish it from the state variable, a single '_' is easy to overlook.

Reply:
@chriseth yeah makes sense; I thought about something like `recipientAddress` for the parameter name or something like that. But then it seemed that using trailing underscore will be more of the current naming guideline.

On it